packages/adapters/wagmi/src/client.ts

```diff
@@ -218,7 +222,7 @@ export class WagmiAdapter implements ChainAdapter {
 
   public construct(appKit: AppKit, options: AppKitOptions) {
     if (!options.projectId) {
-      throw new Error('appkit:initialize - projectId is undefined')
+      return
```

Comment:
Here seems we removed the error throw this time, is it okay to do, how can we warn users?

Reply:
Yeah i think that's fine, we're checking it in the constructor [here](https://github.com/reown-com/appkit/blob/a6d099085dcd23ef90b90153fe0c7199780689fd/packages/adapters/wagmi/src/client.ts#L147-L149).